metron-deployment/roles/kibana/README.md

```diff
@@ -1,35 +0,0 @@
-Kibana 4
---------
-
-This role installs Kibana along with the default Metron Dashboard.
-
-### FAQ
-
-#### How do I change Metron's default dashboard?
```

Comment:
Oh, hey, thanks! Maybe at the end of the deployment readme? I dunno, weirdly enough, I really struggled with a sensible place to put it. Co-located with the module seemed too buried (though, I hope you found the command line help, well, helpful) and way up in deployment seemed mis-located. I was about to put a wiki post together, but then there was some talk about not having one...
Anyway, maybe @ottobackwards has a suggestion, he pointed out the lack of documentation in the first place.